tools/cldr-apps/js/src/views/AboutPanel.vue

```diff
@@ -15,8 +17,23 @@
           <td class="aboutKey">
             {{ k }}
           </td>
-          <td class="aboutValue">
-            {{ v }}
+          <td>
+            <span v-if="/^(http|https):.*/.test(v)">
+              <!-- Looks like a URL -->
+              <a class="aboutValue" v-bind:href="v">{{ v }}</a>
+            </span>
+            <span v-else-if="/^[a-fA-F0-9]{6,40}$/.test(v) && aboutData.CLDR_COMMIT_BASE">
+              <!-- Looks like a Git hash, 6…40 hex chars.
+                   We need to check that CLDR_COMMIT_BASE is set, because that’s the
+                   base URL for the linkification -->
+              <a class="aboutValue" v-bind:href="aboutData.CLDR_COMMIT_BASE + v">
+                <i class="glyphicon glyphicon-cog" />{{ v }}
+              </a>
+            </span>
+            <span v-else class="aboutValue">
+              <!-- some other type -->
+              {{ v }}
+            </span>
           </td>
```

Comment:
wow!

Reply:
do you like this kind of a fix?

Reply:
I like it as a fun new way of doing things, and it seems fine in this context. It's good to know we can do this. When it starts getting too complicated, we might put more of the logic in `<script>` and less in `<template>`. The repetition of `span` and `aboutValue` would get cumbersome if we kept adding more `v-else-if`. Maybe a compromise would be like `v-if="shouldBeLink(v)" ... v-bind:href="makeLink(v)"` -- then there would be v-if and v-else but not v-else-if, and the details of `shouldBeLink` and `makeLink` would be in `<script>`

Reply:
the td's content could be pulled out as a component, kind of a text-or-link object. 

i agree, it could be pulled out to a script - if it was really going to be reused